FiskalySDK/Classes/FiskalyClient/FiskalyClient.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import FiskalySDK.Client
+
+func fiskalyClientInvoke(_ request: String) -> String {
+    let resultRaw = _fiskaly_client_invoke(request)
+    let result = String(cString: resultRaw!)
```

Comment:
I am not sure if a "connection loss" to a framework can happen. Basically you will always get a non-null response, but for the sake of security I will change this too. Thanks for the heads up.

Reply:
I mean, in the end the hosting app will use the framework and for a mobile device connection losses are pretty common. If you absolutely sure it will never ever be nil, feel free to keep though.

Reply:
I am not trying to argue against changing it. I just want to tell you some details so you can understand it better, what may help you when implementing. 
Connection Loss will always end in a Timeout Exception.